### ISSUE TITLE
Aggregate forms > "virtual form" or "multiple custom forms for a help topic" 

### ISSUE THREAD

**Issue by jeffchulg** [osTicket/osTicket-1.7#859]
Hi, I'm facing a problem in parametrization : it's not a bug, but it's quite ennoying.
- when I create a custom form "database creation", I set a list of form fields as "RDBMS", "Version", "Server", etc.

If the help topic is just "Create DB on existing host", it's OK.
But if the help topic is "Create DB on new server", I'll need to create a new custom form (exact copy of the first one) and add additional informations needed for this, which could be actually the custom form for help topic "New Server".

So, basically, if there were a mecanism which allows to create a custom form with multiple parents OR add multiple forms to a help topic, it would be very good for that purpose.
